packages/fiori/src/Timeline.js

```diff
@@ -96,6 +120,19 @@ class Timeline extends UI5Element {
 
 		this._itemNavigation.setCurrentItem(target);
 	}
+
+	onAfterRendering() {
+		this._itemNavigation.NavigationMode = this.axisOrientation === TimelineLayout.Horizontal ? NavigationMode.Horizontal : NavigationMode.Vertical;
```

Comment:
Should not this be `this._itemNavigation.navigationMode` (**n**avigationMode with small "n")

Reply:
I think the whole logic in onAfterRendering can be moved to onBeforeRendering, did you try it?

Reply:
Done